histomicstk/features/ComputeNucleiFeatures.py

```diff
@@ -167,6 +180,26 @@ def ComputeNucleiFeatures(im_label, im_nuclei, im_cytoplasm=None,
 
         feature_list.append(fgrad_cytoplasm)
 
+    # compute nuclei haralick features
+    if haralick_features_flag:
+
+        fharalick_nuclei = ComputeHaralickFeatures(im_label, im_nuclei,
```

Comment:
Ok i like the idea of following that convention about intensity ranges (I will change the graycomatrixext function accordingly).

If we keep [0, 255] as the default, should we increase `num_levels` to something like 32. If we set it to 8, i feel all nuclear pixels will be restricted to few levels making glcm matrix very sparse. 

@slee172 Maybe this is a small experiment we may want to run - compute glcm matrix for say 100 randomly selected nuclei with different values of num_levels (8, 16, 32, 64, 128), and look at the sparsity (1 - fraction of non-zero values) of the glcm. The less sparse the better. And that will tell us what would be a good default value.